### PR TITLE
Add addon designs

### PR DESCRIPTION
**Pull request by josepjaume**
Adds the design addons to link storybook with Figma.

### CODE REVIEW

**Review by sauldom102 — APPROVED** [review]
Supeeer!! 🔥

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           node-version: "21.x"
           cache: npm
-      - run: npm ci
+      - run: npm i
```
Comment (nlopin) [inline .github/workflows/chromatic.yml:27]: why? is it a temporary thing?

**Comment by nlopin** [comment]
is there a way to add a link to the docs page? We don't have a lot of text on the `Button` page, however there is text on the typography page. It would be great to have an way to have a link there

**Comment by josepjaume** [comment]
@nlopin Yeah... I think we can modify the default component to render pages and read from `parameters`. In case there's a `design` entry, we can just render a link :)

**Pull request merged**